doc/examples/example_symm.md

```diff
@@ -57,13 +57,20 @@ assert(std::equal(plaintext.begin(), plaintext.end(), decryptedtext.begin()));
 
 Authenticated ciphers not only encrypt data but also compute authentication tag over it. It allows to detect if data was modified after it was encrypted.
 
+It is also possible to associate additional unencrypted data with the message. The authentication tag will ensure the integrity of both encrypted and unencrypted data in that case.
```

Comment:
Please remove the issue reference from the commit. Since this is open source code, these references are useless to everybody not in our network.

Reply:
Done